apps/homework/templates/homework/overview.html

```diff
@@ -245,7 +252,12 @@ <h4 class="ui header">Submission Details:</h4>
                                                     {% if definition.jupyter_notebook_enabled %}
                                                         <div class="ui tiny statistic">
                                                             <div class="value">
-                                                                {{ grade.jupyter_notebook_grade }}/{{ definition.jupyter_notebook_highest }}
+                                                                {% if not grade.jupyter_notebook_grade %}
```

Comment:
Should be able to do this, may have to make `0.0` a string:
```
{{ grade.jupyter_notebook_grade|default:0.0 }}
```

https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#default

Reply:
✅ Good call